lib/car.rb

```diff
@@ -0,0 +1,17 @@
+class Car
+  attr_reader :make, :model, :monthly_payment,
+              :loan_length, :total_cost, :color
+
+  def initialize(car, monthly_payment, loan_length)
+    @make = car.split(" ").first
+    @model = car.split(" ").last
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+    @total_cost = @monthly_payment * @loan_length
```

Comment:
The only thing I might do is put this in a method rather than an instance var perhaps because it is a calculation. Not a hard and fast rule here though and this does work

```
def total_cost
  @monthly_payment * @loan_length
end
```